### DISCUSSION TITLE
Ideas / Feature Requests

### DISCUSSION THREAD

**Discussion by memstechtips** [memstechtips/WIMUtil#4] [Ideas]
This thread hosts all the ideas or feature requests for WIMUtil.

# Ideas / Feature Requests
- Option to Add Windows Updates to Windows Installation Media

**Comment by ppetrix** [comment]
Can you please remove the security questions from the unattended file? Tnx

**Reply by SheepCommander** [reply]
Don't create a password! If you just don't enter anything for your password, it doesn't show security questions.

**Reply by ppetrix** [reply]
LOL ... ok. tnx

**Reply by SheepCommander** [reply]
(you leave the password field completely blank, press enter, then set your password *after* windows is already setup)

**Comment by KikoPavan** [comment]
How do I install Windows 11 Pro, not Home? I read that it is possible, I am not a programmer and I did not have the confidence to make changes.
